### PR TITLE
Partial revert of pundit cleanup

### PR DESCRIPTION
**Pull request by bgeuken**
Apparently my previous commit (fc126da42483a23fe3d8ff4cc59) was breaking the attribute permission checks in the webui. This PR reverts the faulty commit and adds a test to cover the problem in the future.

### CODE REVIEW

**Comment by bgeuken** [comment]
@evanrolfe Please review

File: src/api/app/policies/attrib_policy.rb [hunk 1]
```diff
@@ -3,11 +3,14 @@ def create?
     # Admins can write everything
     return true if @user.is_admin?
 
-    # No specific rules set for the attribute, check if the user can modify the container
-    return true if @record.container.try(:can_be_modified_by?, @user)
-
-    # check for modifiable_by rules
-    permissions_for_modifiables?(modifiables)
+    at_modifiables = modifiables
```
Comment (evanrolfe) [inline src/api/app/policies/attrib_policy.rb]: Only a suggestion - up to you if you want to implement it or not but you can delete this line if you change memoize the `modifiables` method like this:
```ruby
  def modifiables
    @modifiables ||=
      @record.attrib_type.try(:attrib_type_modifiable_bies).to_a |
        @record.attrib_type.try(:attrib_namespace).try(:attrib_namespace_modifiable_bies).to_a
  end
```

**Comment by evanrolfe** [comment]
LGTM (when travis passes)

File: src/api/app/policies/attrib_policy.rb (hunk 1 above)
Comment (bgeuken) [inline src/api/app/policies/attrib_policy.rb]: That's a good idea. But for now I want to get the regression fixed asap.

I'll get back to this one when the other sprint card got fixed. And then I am going to try your suggestion:-)

**Pull request merged**